src/scroll.cc

```diff
@@ -35,6 +35,51 @@
 #include "x11.h"
 #include <vector>
 
+/**
+ * Length of a character in bytes.
+ * @param c first byte of the character
+ */
+inline int scroll_character_length(char c) {
+#ifdef BUILD_X11
```

Comment:
(see answer to comment above)

Scroll still will work in console mode, but without UTF-8 support.